src/firefly/js/ui/tap/TableSearchMethods.jsx

```diff
@@ -488,7 +488,7 @@ function TemporalSearch({cols, columnsModel, groupKey, fields, useConstraintRedu
         const timeOptions = [{label: 'ISO', value: ISO},
                              {label: 'MJD', value: MJD}];
         const crtTimeMode = get(fields, [TimeOptions, 'value'], ISO);
-        const icon = crtTimeMode === ISO ? 'calendar' : '';
+        const icon = 'calendar'; //display calendar for both ISO and MJD like we do for Exposure Duration Search (ObsCore)
```

Comment:
instead of keeping the extra variable, just remove this line and pass `'calendar'` on line 509 and 523